Sources/RayTracer/RayTracer.Imaging/Png/PngChunk.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace RayTracer.Imaging.Png;
+
+public ref struct PngChunk
```

Comment:
the struct should be readonly, `byte[] Data` would then become `ReadOnlySpan<byte>`